src/sinks/honeycomb.rs

```diff
@@ -24,9 +24,12 @@ pub struct HoneycombConfig {
     endpoint: String,
```

Comment:
Should we document the endpoint format? Looks like it should be a http/https endpoint.

Reply:
😆 turns out we didn't render this before...

Reply:
Ah, additional digging - this isn't user facing and only exists for internal testing. I've added a comment expressing such.